src/components/CornerCircularLabel/CornerCircularLabel.scss

```diff
@@ -0,0 +1,36 @@
+.corner-circular-label {
+  position: absolute;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  border-radius: 50%;
+  width: 3.5vmin;
```

Comment:
A good practice is to create a "grid unit", which in your case would be `1vmin` and to keep it as a variable for reuse.

Reply:
Will keep that in mind. Don't want to eject the project so for now will change all non-screen relative units to screen relative ones and use vmin as my grid unit.